src/Administration/Resources/app/administration/src/app/store/index.ts

```diff
@@ -10,8 +10,7 @@ import {
 
 /**
  * @sw-package framework
- * @private
- * @deprecated: tag:v6.7.0 - Will be public
+ * @public
```

Comment:
the deprecation says this should be public, but now this throws an eslint error, what do we need to do here?

Reply:
yep exactly, thx added the ignore, will now wade through all the pipeline issues